doc/linters.md

```diff
@@ -377,6 +378,30 @@ With the configuration above:
 Use edn/read-string instead of read-string
 ```
 
+### Discouraged namespace
+
+*Keyword*: `:discouraged-namespace`
+
+*Description:* warn on the require or usage of a namespace that is discouraged to be used.
+
+*Default level:* `:warning`
+
+*Config:*
+
+```clojure
+{:linters {:discouraged-namespace {clojure.java.jdbc {:message "Use next.jdbc instead of clojure.java.jdbc"}}}}
+```
+
+The matching namespace symbol may be given a group name using a regex pattern.
```

Comment:
This should probably be updated by mentioning `:config-in-ns`

Reply:
fixed